src/assets/styles/layout.scss

```diff
@@ -52,6 +52,10 @@
   }
 }
 
+.section-container {
+  padding: 1.25rem 1.875rem;
+}
```

Comment:
Fortunately this SCSS file is only used while the components are running locally (Storybook). When imported, the components use the parent's (project's) assets.

Otherwise I'd worry this is changing other components unexpectedly.

Reply:
Definitely! Why i wasn't too worried about adding it in for the stories sake.